package.json

```
@@ -36,10 +35,10 @@
     "build": "pika build",
     "coverage": "tap --coverage-report=html && open coverage/lcov-report/index.html",
     "generate-known-webhooks": "node scripts/update-known-events.js",
-    "lint": "prettier --check 'src/**/*.{js,json}' 'test/**/*' 'scripts/**/*' index.d.ts README.md package.json",
-    "lint:fix": "prettier --write 'src/**/*.{js,json}' 'test/**/*' 'scripts/**/*' index.d.ts README.md package.json",
-    "pretest": "npm run -s lint",
-    "test": "tap --100 --coverage 'test/**/*-test.js'",
+    "lint": "prettier --check 'src/**/*.{ts,json}' 'scripts/**/*' 'test/**/*' README.md package.json",
+    "lint:fix": "prettier --write 'src/**/*.{ts,json}' 'scripts/**/*' 'test/**/*' README.md package.json",
+    "pretest": "npm run -s lint; npm run build -s",
```

```suggestion
    "pretest": "npm run -s lint & npm run build -s",
```

otherwise the script continues, even when `npm run -s lint` fails